script/hooks/keyboards-build-success.php

```diff
@@ -36,15 +37,27 @@ function build_log($message) {
   }
 
   require_once('../../tools/db/build/build.inc.php');
+  require_once('../../tools/db/build/cjk/build.inc.php');
   require_once('../../tools/db/build/datasources.inc.php');
 
-  $DBDataSources = new DBDataSources();
+  function Build() {
```

Comment:
This script had not been updated to support swappable databases. I have updated it to support swappable schemas.